Sources/MessagingInApp/Gist/Managers/Models/Message.swift

```diff
@@ -18,7 +18,7 @@ public class GistProperties {
 
 public class Message {
     public private(set) var instanceId = UUID().uuidString.lowercased()
-    public let queueId: String?
+    public let queueId: String? // uniquely identifies an in-app message in the backend system
```

Comment:
based on the comment beside it, shouldn't this be a message id? the comment says it identifies the message but the property name is queue id